### ISSUE TITLE
[Request]  ZyXEL PK5001Z Modem - Backdoor Account 

### ISSUE THREAD

**Issue by its0x08** [reverse-shell/routersploit#343]
ZyXel exploit request!
Link to exploit: https://www.exploit-db.com/exploits/43105/

**Comment by VegetableCat** [comment]
hi, I just want to write a module for this, but I saw #297 , they had already implemented. see [here](https://github.com/reverse-shell/routersploit/tree/master/routersploit/modules/creds)

**Comment by its0x08** [comment]
idk if u notice but i asked 10days ago..
so at the time there was nothing 🤣

**Comment by lucyoa** [comment]
We changed a little bit approach for default/hardcoded credentials. These kind of modules will be accepted if they cover default creds for specific device/vendor. So modules like:

* zyxel_telnet_default_creds 
* zyxel_ssh_default_creds
* zyxel_web_interface_default_creds
* etc

These modules should just test for a few default creds e.g. 5 of them. The problem that we encountered before was that these device are poorly written and running bruteforce attack against them just crashes services or very often lock out target account.
